plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/reporter/ReporterType.kt

```diff
@@ -0,0 +1,10 @@
+package org.jlleitschuh.gradle.ktlint.reporter
+
+import net.swiftzer.semver.SemVer
+
+enum class ReporterType(val reporterName: String, val availableSinceVersion: SemVer, val fileExtension: String) {
+    plain("plain", SemVer(0, 9, 0), "txt"),
```

Comment:
I agree

Reply:
But then it won't work with Groovy parsing for:
```
reporters = ["plain", "checkstyle"]
```

Reply:
How does groovy figure out which enum to map to? I'm sure there's a method you can implement where you could do something like `equalsIgnoreCase` and it would work fine.

Reply:
@JLLeitschuh this is my first contribution to gradle plugin (and it's written in kotlin), so I have no idea, how to do it. I spent couple hours debugging, how it works etc.